src/uproot/compression.py

```diff
@@ -88,11 +88,15 @@ class _DecompressZLIB:
     name = "ZLIB"
     _2byte = b"ZL"
     _method = b"\x08"
+    zlib = uproot.extras.zlib()
```

Comment:
ZLIB wasn't in extras before because it's part of the Python standard library; we can assume it's always there.